### ISSUE TITLE
Template tags render blank content

### ISSUE THREAD

**Issue by sadmansh** [antiproblemist/django-accountkit#6]
Hello. 

I tried installing django-accountkit for my project, and followed all the steps in the README.md file, but when I use {% accountkitjs %} inside my html file, the whole page is rendered blank: the content inside the <body> tag is deleted.

Here is my register.html file:
```
{% load accountkit %}
{% load staticfiles %}
{% load compress %}

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
	<title>{% block title %}Register | Dorkar.co{% endblock %}</title>
	{% compress css %}
	<link rel="stylesheet" type="text/x-scss" href="{% static 'scss/main.scss' %}">
	{% endcompress %}
	{% accountkitjs %}
</head>
<body class="{% block bodyclass %}{% endblock %}">
	<header>
		<h1 class="site-title">Register</h1>
	</header>
	<main>
		{% accountkitform %}
		{% smslogin %}
	</main>
</body>
</html>
```

views.py:
```
from django.views.decorators.csrf import csrf_exempt
from accountkitlogin.views import login_status

@csrf_exempt
def login_success(request):
	context = login_status(request)
```

urls.py:
```
from . import views
from django.views.generic import TemplateView
from django.urls import path

urlpatterns = [
	path('success/', views.login_success, name='login_success'),
	path('register/', TemplateView.as_view(template_name='register.html')),
]
```

and settings.py:
```
INSTALLED_APPS = [
    ...,
    'accounts',
]
APP_ID = "xxx"
ACCOUNT_KIT_APP_SECRET = "xxx"
ACCOUNT_KIT_VERSION = "v1.0"
ACCOUNT_KIT_SUCCESS_REDIRECT = "http://localhost:8000/success"

AUTHENTICATION_BACKENDS = (
    'accountkitlogin.authenticate.GetOrCreateUser',
)
```

Any idea what I am doing wrong here? Thanks in advance.

**Comment by antiproblemist** [comment]
Have you added `accountkitlogin `to your `INSTALLED_APPS`?

**Comment by sadmansh** [comment]
Yes.

**Comment by antiproblemist** [comment]
It looks like there is `</script>` tag missing in the `{% accountkitjs %}` template tag. For the time being please add the `</script>` tag after the template tag like this `{% accountkitjs %}</script>`. This will be fixed soon

**Comment by antiproblemist** [comment]
This issue has been fixed in django-accountkit 0.2.3

**Comment by sadmansh** [comment]
Thank you!
